cmd/jaeger/config-kafka-ingester.yaml

```diff
@@ -12,6 +12,12 @@ service:
       level: debug
 
 extensions:
+  healthcheckv2:
+    use_v2: true
+    http:
+      endpoint: 0.0.0.0:14133
+    grpc:
+      endpoint: 0.0.0.0:14132
```

Comment:
ingester is already using non-standard metrics port, so I changed it to use non-standard health check ports too